src/pages/AddResort.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const AddResort = () => {
```

Comment:
- [ ] On your Rails app, your included an Admin user:

![image](https://github.com/Thinus01/Resort_Booking_Front-end/assets/57421823/8792186b-91f2-4942-b520-41c4f8603286)

However, it can't login:

![image](https://github.com/Thinus01/Resort_Booking_Front-end/assets/57421823/07c5b861-24ca-4277-88a6-0b91b2973d02)

Kindly, include at least one Admin user with credentials so we can test the ADD and DELETE functionality. 👍

Reply:
@lawrahkonwea Thanks for your valuable suggestion, However, I don't think I should add a dummy admin user to my production database This seed file is only for the development environment but still you can archive it with third-party tools like `Postman`